EllisWeb.Gematria/Calculator.cs

```diff
@@ -167,50 +168,116 @@ public static long GetNumericGematriaValue(string sourceString, GematriaType gem
         /// A value of 15 will always be represented as ט"ו and 16 will be represented as ט"ז, following Jewish custom. 
         /// </remarks>
         /// <returns>Gemtria Numeric representation of given number</returns>
+        [Obsolete("Please use the other overload - ConvertToGematriaNumericString with GematriaOptions")]
         public static string ConvertToGematriaNumericString(long number, bool includeSeparators = true, char thousandsSeparator = '\'', char tensSeparator = '"')
+        {
+            return ConvertToGematriaNumericString(number, new GematriaOptions(){ 
+                IncludeSeparators = includeSeparators,
+                ThousandsSeparator = thousandsSeparator,
+                TensSeparator = tensSeparator
+            });
+        }
+
+        /// <summary>
+        /// Convert a number into its Gematria Numeric representation
+        /// </summary>
+        /// <param name="number">The non-negative number to evaluate</param>
+        /// <param name="options">Gematria conversion options</param>
+        /// <example>
+        /// 8 ==> ח
+        /// 15 ==> ט"ו
+        /// 245 ==> רמ"ה
+        /// 5,767 ==> ה'תשס"ז
+        /// 1,024,999 ==> א'כד'תתרצ"ט
+        /// </example>
+        /// <remarks>
+        /// Will evaluate each thousands-grouping separately, inserting separators if needed.
+        /// A value of 15 will always be represented as ט"ו and 16 will be represented as ט"ז, following Jewish custom. 
+        /// </remarks>
+        /// <returns>Gemtria Numeric representation of given number</returns>
+        public static string ConvertToGematriaNumericString(long number, GematriaOptions options)
```

Comment:
Should change this to 

`public static string ConvertToGematriaNumericString(long number, GematriaOptions options = null)`

And then the first line should be `options = options ?? new GematriaOptions();`

That way it is not required to give options, and if you dont, it will use the defaults.